apps/axmol/src/bim/axmol/app/screen/online_game.cpp

```diff
@@ -635,6 +642,9 @@ void bim::axmol::app::online_game::display_players()
           {
             bim::axmol::app::player& w = *m_players[player.index];
 
+            if (player.invisible && player.index != m_local_player_index)
+              return;
```

Comment:
In games with three players or more, the defeated players can still watch the game of the remaining players. It would be nice if they could still see the invisible players, to avoid staring at an empty screen. In `fog_display.cpp` we check if the player is still alive and change the opacity of the assets accordingly. The same idea would be nice here. Something like: `if (player.invisible) w.setOpacity(local_player_still_alive ? 0 : 127)`.